server/odc-service/src/main/java/com/oceanbase/odc/service/db/DBTableService.java

```diff
@@ -137,10 +144,16 @@ public GenerateTableDDLResp generateCreateDDL(@NotNull ConnectionSession session
 
     public GenerateTableDDLResp generateUpdateDDL(@NotNull ConnectionSession session,
             @NotNull GenerateUpdateTableDDLReq req) {
-        String ddl = session.getSyncJdbcExecutor(
-                ConnectionSessionConstants.BACKEND_DS_KEY)
-                .execute((ConnectionCallback<String>) con -> getTableExtensionPoint(session).generateUpdateDDL(con,
-                        req.getPrevious(), req.getCurrent()));
+        String ddl;
+        if (ConnectionSessionUtil.getLogicalSession(session)) {
+            ddl = DBAccessorUtil.getTableEditor("4.0.0", session.getDialectType())
```

Comment:
Sorry, I didn't understand your point. Does `DBAccessorUtil` belong to the implementation of the plugin?

Reply:
yes, you can check the package of the `DBAccessorUtil`: `com.oceanbase.odc.plugin.schema.obmysql.utils`

Reply:
OK. BTW. If we don't want this class to be relied upon odc-server, shouldn't it not be public?

Reply:
it is not about the specific method, it is about the plugin.

we should use plugin by using its api rather than its implementation. in another word, we should not import the plugin ob-mysql and should not use the DBAccessorUtil, because it is belong to a specific implementation of a plugin